src/data/faq_de.json

```diff
@@ -467,20 +467,26 @@
                                 "anchor": "eu_dcc_export",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Aktualisiert am ??. ??? ????</b><hr>",
                                     "<b>Zweck</b>",
                                     "Ab der Version 2.10 der Corona-Warn-App können Sie eingescannte Digitale COVID Zertifikate der EU in PDF-Dokumente exportieren. Diese Funktion dient vorrangig der Erstellung von Sicherheitskopien der Zertifikate für Ihre persönlichen Zwecke.",
+                                    "Ab der Version 2.24 der Corona-Warn-App können Sie in einem Schritt alle in Ihrer App vorhandenen Impf- und Genesenenzertifikate, sowie alle Tests, die nicht älter als 72 Stunden sind, in einem gemeinsamen PDF-Dokument speichern.",
                                     "Selbstverständlich werden für das Layout der Zertifikate die Original-EU-Vorlagen benutzt.",
                                     "<b>Sicherheitshinweise</b>",
                                     "Der Export und die Speicherung der Zertifikate sind freiwillig. Das Erstellen der PDF-Datei findet ausschließlich lokal auf Ihrem Smartphone statt. Nur Sie selbst haben Zugriff auf dieses Dokument.",
                                     "Beachten Sie aber, dass das erzeugte Dokument sensible persönliche Informationen enthält. Gehen Sie daher bitte sorgsam mit der PDF-Datei um. Zeigen Sie das Dokument nur solchen Personen vor, denen Sie vertrauen und die zur Prüfung auch berechtigt sind.",
                                     "Wir raten dringend davon ab, das PDF-Dokument zu veröffentlichen über potentiell unsichere Kanäle (z.&nbsp;B. per E-Mail oder über andere Apps) zu teilen.",
                                     "<b>Limitierung</b>",
                                     "Mit der neuen Funktion können nur PDF-Dokumente von Zertifikaten erzeugt werden, die auch in Deutschland ausgestellt wurden. Ansonsten erscheint eine ausführliche Meldung: 'Druckversion anzeigen nicht möglich - Die Druckversion des Zertifikats kann nicht angezeigt werden, da das Zertifikat nicht in Deutschland ausgestellt wurde.'",

```

Comment:
"Beim Export einzelner Zertifikate ..." instead of "Mit der neuen Funktion"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2961/commits/7e085cdd9b7b63f28941b34fb0eacc970ebf8d85, thanks for your suggestion